### DISCUSSION TITLE
3章15節「HTMLのフォーム」のコーディング

### DISCUSSION THREAD

**Discussion by satsukizzz** [satsukizzz/n-playspace-http#14] [Show and tell]
こちらは3章14節 https://github.com/satsukizzz/n-playspace-http/discussions/12 が終わってからコーディングできると思います。
そちらがマージされた後のコミットから開発を始めてください。

コーディングする方を募集しています 🥣

**Comment by HappyKumamoto** [comment]
電卓の開発が開始されました！
こちらの開発も急がなければいけない気がします。
私は3章15節の練習問題を担当させていただくことになっていますが、
15節本編 + 練習問題 で、3-15 という名前のブランチにして、
ひとつのプルリクにして出しても良いでしょうか。

もし、許可していただけるとしたら、にょろぼんさんの14節がマージされ
次第出せるように準備したいと思います。よろしくお願いします。

**Reply by satsukizzz** [reply]
検討ありがとうございます。
コード以外の部分を学ぶだけの余裕が生まれるように、すでにやっていて自信の持てるような教材のプルリクをあげてもらってました。ですので急がなくて大丈夫です！ほかの方の練習用にも練習問題を残してもらって頂いたほうが嬉しい 👀 

ちなみに、チーム開発でコードをどこに書いたらいいのかわからないという場合、オーナーや上司、つまりは管理者に聞くのがいいんです～。電卓のが入ってきてわけがわからないと思うことのないよう、そのあたりはサポートします！
また、電卓のほうでコード書きを含めた挑戦をしてもらってもいいです（けど、HappyKumamotoさんは教材で一旦終わりにするのでしたかね ）。

**Reply by HappyKumamoto** [reply]
わかりました！
学びの場ということを忘れて、勝手に焦ってしまっていました。💦

それでは、15節のコーディングを考えていたという方がいらっしゃらなければ、
私が15節本編を担当させていただき、当初申し出ておりました15節の練習問題はキャンセルと
させていただきたいと思います。

>HappyKumamotoさんは教材で一旦終わりにするのでしたかね

そうなんです。
私は教材でのプルリク練習に入れていただけた経験で充分です。
本当にありがたいと思っています。
教材の部分が修了した段階で、collaborator も解除でお願いいたします。🙇

**Comment by HappyKumamoto** [comment]
ご無沙汰してすみません。
そして、自分から申し出ておきながら申し訳ありませんが、15節のコーディングをキャンセルさせてください。
ごめんなさい🙇

**Reply by satsukizzz** [reply]
了解です！
